Rakefile

```diff
@@ -71,6 +71,18 @@ task(:rubocop) do
   sh "util/rubocop"
 end
 
+desc "Run a test suite bisection"
+task(:bisect) do
+  seed = begin
+           Integer(ENV["SEED"])
```

Comment:
I'd vote to let the task take an argument instead of relying on an ENV setting. Or you could do both:

```
task(:bisect, [:seed]) do
   seed ||= ENV['SEED']
   # ...
end
```
Then call it like `rake bisect[12345]`.

Reply:
That's neat, I'll add that!

Reply:
I was about to add this but I'm now not really sure I prefer it. Given that for running tests we do `SEED=12345 rake test`, but we can't do `rake test[12345]`. Doesn't it make more sense to keep it consistent?

Reply:
Well, as I said, you can support either approach by defaulting to ENV['SEED'] if no arg is provided, so CI would still work. But, it's your PR. :)